src/renderer/components/attachment/Attachment.tsx

```diff
@@ -0,0 +1,83 @@
+import { ipcRenderer } from 'electron'
+import mimeTypes from 'mime-types'
+
+/* Section - Data Copied in part from Signal */
+// Supported media types in google chrome
+// See: https://en.wikipedia.org/wiki/Comparison_of_web_browsers#Image_format_support
+const SUPPORTED_IMAGE_MIME_TYPES = Object.freeze([
+  'image/bmp',
+  'image/gif',
+  'image/jpeg',
+  'image/jpg',
+  // No need to support SVG
+  //'image/svg+xml',
+  'image/webp',
+  'image/x-xbitmap',
+  // ICO
+  'image/vnd.microsoft.icon', 'image/ico', 'image/icon', 'image/x-icon',
+  // PNG
+  'image/apng', 'image/png'
+])
+// See: https://www.chromium.org/audio-video
+const SUPPORTED_VIDEO_MIME_TYPES = Object.freeze([
+  'video/mp4',
+  'video/ogg',
+  'video/webm'
+])
+/* EndSection - Data Copied in part from Signal */
+
+// TODO define this correctly 
+// (maybe inside shared module??, but that depends on wether its also used in the backend or just exists in the frontend)
+export type attachment = { [key: string]: any, contentType: string, fileName: string, url: any }
```

Comment:
Most other type definitions are split over multiple lines (one per attribute). I find that easier to read, could you adapt this line, too?

Also what about the TODO?

Reply:
that's a stub, and also a incomplete file. That type will later get defined closer to where its created and then imported here so I didn't do it pretty here to save lines. But If you absolutely need that for it to be merged I can split it into multiple line, of course.

Reply:
I think we should write code under the assumption that it will live longer than expected. Quick&dirty leads to badly maintainable code, like DC Desktop already has a lot.
But I won't insist on this, no.

Reply:
nah its already better than just writing `any` everywhere